ONSdigital/dp-python-tools
# Added SesClient and tests

### What

Added python client for sending emails using aws ses and the boto3 client. 
And added tests and ses to the environment

### How to review

- Run all the test for test_email.py
- Check the test logic is correct
- check for the SesClient logic is correct
- Any typo in documentation or comments

### Who can review

Anyone
